db/schema.rb

```diff
@@ -10,14 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_10_30_111940) do
+ActiveRecord::Schema.define(version: 2020_12_11_160500) do
```

Comment:
Please review, this is my first conflict resolutions for `schema.rb`.

Reply:
Looks good, most importantly the date is earlier. Maybe the following link helps also: https://stackoverflow.com/questions/2979059/rails-is-the-version-number-in-schema-rb-used-for-anything